wordpress-mobile/WordPress-iOS
# Reader: Update tag recommendation card

Refs #21646 

As titled, this updates the design of the "You might like" cards.

Light | Dark
-|-
![Simulator Screenshot - iPhone 15 - 2023-10-14 at 07 12 43](https://github.com/wordpress-mobile/WordPress-iOS/assets/1299411/fcca0e0a-738b-481b-81f4-e268d1a83da0) | ![Simulator Screenshot - iPhone 15 - 2023-10-14 at 07 13 29](https://github.com/wordpress-mobile/WordPress-iOS/assets/1299411/118acced-9d71-4de9-b6bc-6ce55600c95d)

## To test

- Launch Jetpack
- Enable the `readerImprovements` feature flag
- Go to Discover tab
- 🔎 Verify that the tag recommendation cards appear correctly.
- Tap on one of the tags.
- 🔎 Verify that it takes you to the tag stream page.

## Regression Notes
1. Potential unintended areas of impact
Should be none. Feature is hidden behind a flag.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the changes.

3. What automated tests I added (or what prevented me from doing so)
N/A.

PR submission checklist:


UI Changes testing checklist:
